test/facets/third-party-facet.spec.ts

```diff
@@ -1,568 +0,0 @@
-import "@nomiclabs/hardhat-waffle";
```

Comment:
Deleted the entire file because we didn't have any other third party facet functions being tested here